docker/service/src/test/java/uk/nhs/adaptors/scr/mappings/from/fhir/CareEventMapperTest.java

```diff
@@ -1,56 +1,46 @@
 package uk.nhs.adaptors.scr.mappings.from.fhir;
 
 import org.hl7.fhir.r4.model.Encounter;
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ArgumentsSource;
 import org.mockito.InjectMocks;
-import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
-import uk.nhs.adaptors.scr.components.FhirParser;
-import uk.nhs.adaptors.scr.mappings.from.common.UuidWrapper;
 import uk.nhs.adaptors.scr.models.GpSummary;
 import uk.nhs.adaptors.scr.models.xml.CareEvent;
 import uk.nhs.adaptors.scr.utils.TemplateUtils;
-import uk.nhs.utils.CareEventMapperArgumentsProvider;
-
 import java.util.ArrayList;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.when;
-import static uk.nhs.utils.Utils.readResourceFile;
 
 @ExtendWith(MockitoExtension.class)
-public class CareEventMapperTest {
+public class CareEventMapperTest extends BaseFhirMapperTest {
 
     @InjectMocks
     private CareEventMapper careEvent;
 
-    @Mock
-    private UuidWrapper uuid;
-
-    private FhirParser fhirParser = new FhirParser();
+    private static final String ID = "0F582D91-8F89-11EA-8B2D-B741F13EFC47";
+    private static final String RESOURCE_DIRECTORY = "care_event";
+    private static final String STATUS_CODE = "normal";
+    private static final String FILE_NAME = "example";
 
-    @ParameterizedTest(name = "[{index}] - {0}.html/json")
-    @ArgumentsSource(CareEventMapperArgumentsProvider.class)
-    public void When_MappingFromFHIR_Expect_RootId(String fileName) {
-        var json = readResourceFile(String.format("care_event/%s.json", fileName));
+    @Test
+    public void When_MappingFromFHIR_Expect_RootId() {
+        var json = getJsonExample(RESOURCE_DIRECTORY, FILE_NAME);
 
-        when(uuid.randomUuid()).thenReturn("0F582D91-8F89-11EA-8B2D-B741F13EFC47");
+        returnExpectedUuid(ID);
 
-        var encounter = fhirParser.parseResource(json, Encounter.class);
+        var encounter = getFhirParser(json, Encounter.class);
```

Comment:
you should be able to do getJsonExample and getFhirParser in one call to base class. Perhaps getFileAsObject(RESOURCE_DIRECTORY, FILE_NAME, Encounter.class) ?